graphs/bfs_shortest_path.py

```diff
@@ -1,15 +1,37 @@
-graph = {
```

Comment:
Please put this back in so that we can define __graph__ once instead of three separate times

Reply:
Makes sense. Thanks, done.